.changeset/loud-rules-count.md

```diff
@@ -0,0 +1,6 @@
+---
+'@clerk/clerk-js': patch
+'@clerk/types': patch
+---
+
+Reverting changes for base64 support in setProfileImage
```

Comment:
Let's remove this changeset all together, as changesets are supposed to describe public changes.
You can create an empty changeset this with `npm run changeset -- --empty`